lib/brewdler/utils.rb

```diff
@@ -0,0 +1,10 @@
+module Brewdler
+  def self.system cmd, *args
+    pid = fork do
+      $stdout.reopen("/dev/null")
+      exec(cmd, *args)
+    end
+    Process.wait(pid)
+    $?.success?
```

Comment:
I reckon either we should make sure we always check the exit codes or raise an exception here. Your call.